README.md

```diff
@@ -11,14 +11,17 @@ Other Flutter platforms are not currently supported by this SDK.
 
 ## Installation
 
-### Pub
+### Flutter CLI
 
-The SDK can be installed into your Flutter project by adding the following to your pubspec.yaml:
+The SDK can be installed into your Flutter project by running `flutter pub add devcycle_flutter_client_sdk`.
+
+### Pub Spec
+
+The SDK can be installed into your Flutter project by adding the following to your `pubspec.yaml`:
 
 ```dart
-devcycle_flutter_client_sdk: ^1.0.0
+devcycle_flutter_client_sdk:
```

Comment:
should we include a placeholder to indicate that they'll need to pin a version?

Reply:
I guess we can leave this as `^1.0.0` here to allow all `1.x` versions? 🤔  then we can update going forward once we hit a `2.0.0` release in the future

Reply:
sure, it's easier to keep the readme up to date with releases